README.md

```diff
@@ -27,63 +27,54 @@ We also have edge agents for other platforms:
 - Agent: A component using all other building blocks, provides basic edge agent capabilities, including implementing DIDComm V2 protocols.
 
 ## Getting started
+We highly recommend you [check out the online docs](https://hyperledger.github.io/identus-docs/docs/getting-started) :world_map:
 
-This repository includes a browser and a Node.js demo application, and also a step-by-step documented process on [how to run it](#running-a-demo-project).
-
-### Use in your project
-You can install and use this library in browsers and nodejs.
+### Install
 
 ```bash
-npm i @hyperledger/identus-edge-agent-sdk --save
+npm i @hyperledger/identus-edge-agent-sdk
 ```
 
-or with yarn
-
+or
 ```bash
 yarn add @atala/prism-wallet-sdk
```

Comment:
thanks for contribution, @mixmix 
I think this like must be removed as all new packages of SDKs are under the Hyperledger org.

Reply:
Good point. That wasn't even my writing ... it's a refactor of straight outdated docs. I'll fix it though